shell/platform/windows/BUILD.gn

```diff
@@ -159,9 +159,13 @@ source_set("flutter_windows_source") {
     defines = [
       "_SILENCE_CLANG_COROUTINE_MESSAGE",
       "FLUTTER_ENGINE_NO_PROTOTYPES",
+      "_SILENCE_CXX17_CODECVT_HEADER_DEPRECATION_WARNING",
     ]
   } else {
-    defines = [ "FLUTTER_ENGINE_NO_PROTOTYPES" ]
+    defines = [
+      "FLUTTER_ENGINE_NO_PROTOTYPES",
+      "_SILENCE_CXX17_CODECVT_HEADER_DEPRECATION_WARNING",
```

Comment:
For these two plus the one below: I think you should be safe to remove these definitions now that all the encoding conversion stuff has moved to FML (I was able to remove it everywhere else).